cmd/kfulint/testdata/big-copy/negative_tests.go

```diff
@@ -0,0 +1,41 @@
+package checker_test
```

Comment:
We renamed `big-copy` checker.
It should be called `range-val-copy` now.

Reply:
Oops, fixed.